oak-run-commons/src/main/java/org/apache/jackrabbit/oak/index/indexer/document/NodeStateEntryTraverser.java

```diff
@@ -176,12 +175,6 @@ private boolean includeId(String id) {
         }
 
         String path = Utils.getPathFromId(id);
-
-        //Exclude hidden nodes from index data
-        if (NodeStateUtils.isHiddenPath(path)){
-            return false;
-        }
-
         return pathPredicate.test(path);
```

Comment:
I think we should also move pathPredicate.test, not only NodeStateUtils.isHiddenPath... right? I mean, assuming there is a path predicate like "/content/project" then we only store the intermediate progress for nodes within "/content/project"... and that could in theory be: no node at all. So on each scale up / scale down, we download from scratch.

Reply:
removed and refactored